README.md

```diff
@@ -322,10 +424,75 @@ key = "CREATED" # options: "CREATED"|"MODIFIED"
 #       - for a more comprehensive list see https://en.wikipedia.org/wiki/List_of_UTC_time_offsets
 zone_offset = None
 ```
-
 Since python datetime objects are zone agnostic, provide the proper timezone offset from UTC
 in zone_offset to search properly.
 
+### Boolean Searching
+
+Within each match or exclude call the internal search terms can either be joined with AND or OR. 
```

Comment:
`each match or exclude` `clause` or `parameter` vs `call`. `can be joined with either an AND or OR. SO MANY BOOLEANS!